programs/client/Client.cpp

```diff
@@ -258,6 +318,16 @@ try
     if (is_interactive && !config().has("no-warnings"))
         showWarnings();
 
+    /// Load password complexity rules
+    try
+    {
+        loadPasswordComplexityRules();
+    }
+    catch (...)
+    {
+        /// Ignore exception
```

Comment:
We need at least write this exception to logs

Reply:
Oh, forgot that we are on the client side... Can we simply rethrow an exception?